Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.7.2'
-
+gem 'bootstrap-sass', '3.4.1'
```

Comment:
Nháy đôi anh nhé
đọc lại coding convention nè

rồi self check toàn bộ file này

Reply:
cho anh hỏi là mình fix rồi pull request lên lại laf thành 1 commit mới hay mionhf gộp commit hay là edit trực tiếp trên đây lun. tk